### ISSUE TITLE
[GLYT] FASA FASH GroundLayout

### ISSUE THREAD

**Issue by RoboFun01** [VATSIM-SSA/sectorfile-fasa#40]
**Are you suggesting a mere update or a new Ground Layout?**

- [x] New Ground Layout
- [ ] Update to an existing Ground Layout

**Description**
A new ground layout for FASH would be very nice, as a lot of VFR traffic depart from FASH.

**Comment by RoboFun01** [comment]
I'm working on this

**Comment by jonohanekom** [comment]
FASH doesn't have any ATC though. It's a non-towered aerodrome.

**Comment by RoboFun01** [comment]
Yup. But, I knew this will only take me like 30 mins, and I thought that it would help tower controllers have better awareness, to be able to see what is going on, on the ground at FASH, and to see which aircrafts are in the air, and which ones are on the ground, although this is really not nessassary. So I have done the work, and I will rather make a pull request, so that MIguel can decide wether it is nessassary.

But, should I rather change this, so that the airport is FACT, so that it displays on the FACT ASR, and then just delete the FASH ASR?

**Comment by RoboFun01** [comment]
Right, I have changed it a bit, meaning now FASH is displayed on FACT ASR, rather than having its own ASR, as it is a non-controlled aerodrome like you mentioned.
